src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -162,78 +164,63 @@ These events are also available for querying, creating custom dashboards, and al
 
-You can create a configuration in the anomaly detection UI:
+  You can create a configuration in the anomaly detection UI:
 
-1. From **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, click **Alerts & AI**.
-2. Under **Anomaly detection**, click **Settings**.
-3. Click **Add a configuration**.
-4. Input the following information into the form:
+  1. From **[one.newrelic.com](https://one.newrelic.com/all-capabilities)**, click **Alerts & AI**.
+  2. Under **Anomaly detection**, click **Settings**.
+  3. Click **Add a configuration**.
+  4. Input the following information into the form:
 
    * Choose a name for your configuration that helps you easily identify it from others in your account.
    * Select an account.
    * Select up to 1,000 applications. Note that certain applications with low throughput might not be good candidates for anomaly detection, as they can be more sensitive to smaller amounts of data fluctuation.
-5. Optional: select the golden signals you'd like to monitor for anomalies.
-6. Optional: [connect to incident intelligence](#add-source).
+  5. Optional: select the golden signals you'd like to monitor for anomalies.
+  6. Optional: [connect to incident intelligence](#add-source).
 
-<Callout variant="tip">
+  <Callout variant="tip">
 
-Workflows will soon be replacing configurations. Learn more about our workflows [here](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/).
+  Workflows will soon be replacing configurations. Learn more about our workflows [here](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/).
   </Callout>
 
-<CollapserGroup>
-  <Collapser
-    id="slack-set-up"
-    title="Set up for Slack"
-  >
-    To use anomaly detection with Slack:
 
-    1. Select **Slack**.
-    2. Choose which Slack channel receives notifications. You can select any existing public or private channel. This prompts the workflow to add the applied intelligence Slack application to your selected channel. To create a new channel, do that directly in Slack first.
-       <Callout variant="tip">If you experience an error when assigning Slack channels, make sure that the New Relic AI Slack application has been [added to your Slack workspace](https://slack.com/help/articles/202035138-Add-an-app-to-your-workspace).</Callout>
-    3. Save the configuration.
+### Use anomaly detection with Slack:
 
-       You can modify the applications for each configuration at any time by selecting the configuration in the configuration table.
-  </Collapser>
+  1. Select **Slack**.
+  2. Choose which Slack channel receives notifications. You can select any existing public or private channel. This prompts the workflow to add the applied intelligence Slack application to your selected channel. To create a new channel, do that directly in Slack first.
```

Comment:
I think the last sentence can be tweaked a bit maybe? Something like "To create a new channel for anomaly detection, create the channel in Slack first, then select that channel." Just something along those lines.